docs/en/connector-v2/sink/Kafka.md

```diff
@@ -30,19 +30,19 @@ They can be downloaded via install-plugin.sh or from the Maven central repositor
-| assign_partitions    | Array  | No       | -       | We can decide which partition to send based on the content of the message. The function of this parameter is to distribute information.                                                                                                                                                                                                                                                                                                   |
-| transaction_prefix   | String | No       | -       | If semantic is specified as EXACTLY_ONCE, the producer will write all messages in a Kafka transaction,kafka distinguishes different transactions by different transactionId. This parameter is prefix of  kafka  transactionId, make sure different job use different prefix.                                                                                                                                                             |
-| format               | String | No       | json    | Data format. The default format is json. Optional text format, canal-json, debezium-json and avro.If you use json or text format. The default field separator is ", ". If you customize the delimiter, add the "field_delimiter" option.If you use canal format, please refer to [canal-json](../formats/canal-json.md) for details.If you use debezium format, please refer to [debezium-json](../formats/debezium-json.md) for details. |
-| field_delimiter      | String | No       | ,       | Customize the field delimiter for data format.                                                                                                                                                                                                                                                                                                                                                                                            |
-| common-options       |        | No       | -       | Source plugin common parameters, please refer to [Source Common Options](common-options.md) for details                                                                                                                                                                                                                                                                                                                                   |
+|         Name         |  Type  | Required | Default |                                                                                                                                                                                                                     Description                                                                                                                                                                                                                     |
```

Comment:
Could you help explain that changing those lines in the doc helps to solve the https://github.com/apache/seatunnel/issues/6632?

Reply:
in https://github.com/apache/seatunnel/blob/dev/seatunnel-connectors-v2/connector-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/config/MessageFormat.java enum, it's `debezium_json`  not `debezium-json`. so throw debezium-json is not supported error